### PR TITLE
[CONTINT-3593] Add nginx kata workload in aks scenario

### PR DESCRIPTION
**Pull request by AliDatadog**
What does this PR do?
---------------------
Add a nginx kata workload.

Which scenarios this will impact?
-------------------
AKS Scenario

Motivation
----------
We need to QA that we get container metrics for kata containers.

Additional Notes
----------------

### CODE REVIEW

**Review by pducolin — APPROVED** [review]

**Comment by pducolin** [comment]
Ok for the scenarios

File: scenarios/azure/aks/run.go [hunk 1]
```diff
@@ -51,14 +53,18 @@ func Run(ctx *pulumi.Context) error {
...
 			customValues := `
 datadog:
   kubelet:
     host:
       valueFrom:
         fieldRef:
-          fieldPath: spec.nodeName
+          fieldPath: status.hostIP
     hostCAPath: /etc/kubernetes/certs/kubeletserver.crt
+	tlsVerify: false
```
Comment (AliDatadog) [inline scenarios/azure/aks/run.go:67]: On some nodes, DNS does not resolve the nodename. The only SAN in the certificate is the node name so we can't use tlsVerify

File: components/datadog/apps/nginx/k8s.go [hunk 2]
```diff
@@ -106,11 +118,13 @@ func K8sAppDefinition(e config.CommonEnvironment, kubeProvider *kubernetes.Provi
 								HttpGet: &corev1.HTTPGetActionArgs{
 									Port: pulumi.Int(80),
 								},
+								TimeoutSeconds: pulumi.Int(5),
```
Comment (AliDatadog) [inline components/datadog/apps/nginx/k8s.go:121]: Failed with 1 second on kata

File: components/datadog/apps/nginx/k8s.go [hunk 3]
```diff
@@ -358,7 +373,7 @@ func K8sAppDefinition(e config.CommonEnvironment, kubeProvider *kubernetes.Provi
 							Resources: &corev1.ResourceRequirementsArgs{
 								Limits: pulumi.StringMap{
 									"cpu":    pulumi.String("100m"),
-									"memory": pulumi.String("32Mi"),
+									"memory": pulumi.String("64Mi"),
```
Comment (AliDatadog) [inline components/datadog/apps/nginx/k8s.go:376]: Was oom killed at 32Mi on kata

**Pull request merged**